packages/block-editor/src/components/block-draggable/index.js

```diff
@@ -5,7 +5,41 @@ import { Draggable } from '@wordpress/components';
 import { useSelect, useDispatch } from '@wordpress/data';
 import { useEffect, useRef } from '@wordpress/element';
 
-const BlockDraggable = ( { children, clientIds, cloneClassname } ) => {
+const SCROLL_INACTIVE_DISTANCE_PX = 50;
+const SCROLL_INTERVAL_MS = 50;
+const PIXELS_PER_SECOND_PER_DISTANCE = 5;
+const VELOCITY_MULTIPLIER =
+	PIXELS_PER_SECOND_PER_DISTANCE * ( SCROLL_INTERVAL_MS / 1000 );
+
+function startScrollingY( nodeRef, velocityRef ) {
+	return setInterval( () => {
+		if ( nodeRef.current && velocityRef.current ) {
+			const newTop = nodeRef.current.scrollTop + velocityRef.current;
+
+			nodeRef.current.scroll( {
+				top: newTop,
+				// behavior: 'smooth' // seems to hurt performance, better to use a small scroll interval
+			} );
+		}
+	}, SCROLL_INTERVAL_MS );
+}
+
+function getVerticalScrollParent( node ) {
+	if ( node === null ) {
+		return null;
+	}
+
+	// @todo - remove this hack and get a real reference to the scroll parent
+	return node.closest( '.interface-interface-skeleton__content' );
```

Comment:
Important to replace this as the block editor can be used without the `.interface-interface-skeleton__content`.

Looks like the DOM package has a `getScrollContainer` function that should hopefully do the job. I found that by checking the `Typewriter` component, which also performs scrolling ... there might also be more reusable code in that component.

Reply:
Great suggestion - thanks!